frontend/common/src/components/PageHeader/pageHeader.css

```diff
@@ -1,4 +1,4 @@
-.page-header__icon {
+/* .page-header__icon {
```

Comment:
Any reason to keep this file if all the selectors are commented out? It is being referenced in: https://github.com/GCTC-NTGC/gc-digital-talent/pull/3051/files#diff-37c55dfd9d4a02fe5ddf45c17ff7a82ea0784721e7eea4c8c1a7f6de3cb8b487L4

Reply:
Nope, no reason anymore

Reply:
- [ ] remove file `frontend/common/src/components/PageHeader/pageHeader.css` and related imports